facebook-common/src/main/java/com/facebook/internal/WebDialog.java

```diff
@@ -129,6 +125,9 @@
     private boolean isPageFinished = false;
     private static volatile int webDialogTheme;
 
+    // Used to work around an Android Autofill bug - see Utility.mustFixWindowParamsForAutofill
```

Comment:
How long will this workaround need to be in here?

Reply:
Unfortunately it needs to stay here for a long time, as it affect devices running Android O / APIs 26 and 27 (i.e, it would be here until the minimum target supported is higher than that.)